### PR TITLE
feat: invite users from project view

### PR DESCRIPTION
**Pull request by rephus**
Closes: #2441
### Description:
Peek 2022-06-22 16-45

### CODE REVIEW

**Comment by ZeRego** [comment]
@rephus Should we show the invite url after clicking 'send invite' ? if they don't have SMTP setup the user won't have a way to create an account.

**Comment by rephus** [comment]
Copied the components + styles from invitesPanel 

Peek 2022-06-23 13-27

**Comment by hamzahc1** [comment]
When I tried to QA this and added a user from the project view: 

- An invite link was generated (but not an email invite?) 
- When invite was accepted, the user was added to the organisation but not to the project.

**Comment by ZeRego** [comment]
> An invite link was generated (but not an email invite?)

@hamzahc1 PR environments don't send emails. 

> When invite was accepted, the user was added to the organisation but not to the project.

Did you click `Give access' ? 

I think we should only create the user + show the invite url after clicking the 'Give access' otherwise it is confusing for the user. We show success messages saying 'user was been added' and the success toasts which makes it seem it has been invited to the project as well.

**Comment by hamzahc1** [comment]
>Did you click `Give access' ?

No I didn't - this really wasn't obvious 😅 . I got a success message and it said xxx@gmail.com had been added so I assumed it was done! 
Screenshot 2022-06-23 at 13 28 44

**Comment by rephus** [comment]
only create the user + show the invite url after clicking the 'Give access' 

Peek 2022-06-23 15-44

**Review by ZeRego — APPROVED** [review]

File: packages/frontend/src/components/ProjectAccess/ProjectAccessCreation/index.tsx [hunk 1]
```diff
@@ -64,19 +90,49 @@ const ProjectAccessCreation: FC<{
         }
     }, [isError, methods, isSuccess, showToastSuccess, setEmailSelected]);
 
+    useEffect(() => {
+        if (isInvitationSuccess) {
+            setInviteLink(inviteData);
+            reset();
+            console.log(
```
Comment (ZeRego) [inline packages/frontend/src/components/ProjectAccess/ProjectAccessCreation/index.tsx:97]: forgotten console

**Pull request merged**